src/routes/exercises/ExercisesScreen.tsx

```diff
@@ -56,10 +58,14 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenProps): JSX.Eleme
     apiKey: discipline.apiKey
   })
 
-  const currentExercise: Exercise = EXERCISES[2] //  TODO: LUN-131 logic , note: currentExercise is the last level that can be accessed.
+  useFocusEffect(() => {
+    getDoneExercises(disciplineId)
+      .then(value => setNextExercise(value < EXERCISES.length ? EXERCISES[value] : null))
+      .catch(reportError)
+  })
```

Comment:
Shouldn't this be wrapped in a `useCallback`?

Reply:
good catch, it should. Docs say:

`Note: To avoid the running the effect too often, it's important to wrap the callback in useCallback before passing it to useFocusEffect as shown in the example.`